### PR TITLE
xsdk: add v1.1.0

### PR DESCRIPTION
**Pull request by balay**

### CODE REVIEW

File: var/spack/repos/builtin/packages/xsdk/package.py [hunk 1]
```diff
@@ -105,16 +106,25 @@ class Xsdk(BundlePackage, CudaPackage, ROCmPackage):
...
+    xsdk_depends_on(
+        "hypre@2.32.0+superlu-dist+shared", when="@1.1.0", cuda_var="cuda", rocm_var="rocm"
+    )
     xsdk_depends_on(
         "hypre@2.30.0+superlu-dist+shared", when="@1.0.0", cuda_var="cuda", rocm_var="rocm"
     )
     xsdk_depends_on("hypre@2.26.0+superlu-dist+shared", when="@0.8.0", cuda_var="cuda")
 
+    xsdk_depends_on(
+        "mfem@4.7.0+shared+mpi+superlu-dist+petsc+sundials+examples+miniapps cxxstd=14 ",
```
Comment (v-dobrev) [inline var/spack/repos/builtin/packages/xsdk/package.py]: The option `cxxstd=14` should not be necessary. In fact, due to the `sundials` dependency, the C++ standard will be set to C++14 automatically.

Comment (balay) [reply var/spack/repos/builtin/packages/xsdk/package.py]: Thanks! pushed this change

File: var/spack/repos/builtin/packages/xsdk/package.py [hunk 2]
```diff
@@ -190,6 +221,12 @@ class Xsdk(BundlePackage, CudaPackage, ROCmPackage):
     xsdk_depends_on("sundials +trilinos", when="+trilinos")
     xsdk_depends_on("sundials +ginkgo", when="+ginkgo @0.8.0:")
     xsdk_depends_on("sundials +sycl cxxstd=17", when="@1.0.0: +sycl")
+    xsdk_depends_on(
+        "sundials@7.1.1~int64+hypre+petsc+superlu-dist",
```
Comment (balay) [inline var/spack/repos/builtin/packages/xsdk/package.py:225]: Will update to sundials@7.2.0 - when available

File: var/spack/repos/builtin/packages/xsdk/package.py (hunk 1 above)
Comment (balay) [reply var/spack/repos/builtin/packages/xsdk/package.py]: BTW: This issue came up with `mfem+hip` so perhaps `cxxstd=auto` should detect hip dependency and set c++14 automatically?

**Comment by balay** [comment]
@v-dobrev @balos1 @luszczek @shuds13 @ulrikeyang , please check and approve this PR so that we can merge in.

File: var/spack/repos/builtin/packages/xsdk/package.py (hunk 2 above)
Comment (balay) [reply var/spack/repos/builtin/packages/xsdk/package.py:225]: This update is deferred

**Review by balos1 — APPROVED** [review]

**Comment by balay** [comment]
A successful pipeline (with a merge to latest develop) at:

https://gitlab.com/xsdk-project/spack-xsdk/-/pipelines/1552891746

**Review by v-dobrev — APPROVED** [review]

File: var/spack/repos/builtin/packages/xsdk/package.py (hunk 1 above)
Comment (v-dobrev) [reply var/spack/repos/builtin/packages/xsdk/package.py]: Why do you think using HIP requires C++14? I think C++11 is sufficient.

Comment (balay) [reply var/spack/repos/builtin/packages/xsdk/package.py]: I'll have to recheck builds - but I think some of the hip/rocm includes require c++14 - and builds fail when built with `std=cxx11` compiler flag.

Comment (balay) [reply var/spack/repos/builtin/packages/xsdk/package.py]: Created an issue for this failure at https://github.com/xsdk-project/xsdk-issues/issues/267

**Pull request merged**